services/user/Webmail/ui/src/layouts/default.tsx

```diff
@@ -6,38 +6,43 @@ import {
     ResizablePanel,
     ResizablePanelGroup,
 } from "@shadcn/resizable";
-import { NavMenu } from "@components";
+import { Toaster } from "@shadcn/sonner";
+
+import { NavMenu, ThemeProvider } from "@components";
 import { cn } from "@lib/utils";
 
 export default function DefaultLayout() {
     const [isCollapsed, setIsCollapsed] = React.useState(false);
 
     return (
-        <div className="h-dvh max-h-dvh">
-            <ResizablePanelGroup
-                direction="horizontal"
-                autoSaveId="main-panels"
-            >
-                <ResizablePanel
-                    id="nav"
-                    order={1}
-                    collapsible={true}
-                    minSize={15}
-                    maxSize={20}
-                    onCollapse={() => setIsCollapsed(true)}
-                    onExpand={() => setIsCollapsed(false)}
-                    className={cn(
-                        isCollapsed &&
-                            "min-w-[50px] transition-all duration-300 ease-in-out",
-                    )}
+        <ThemeProvider defaultTheme="dark" storageKey="vite-ui-theme">
```

Comment:
Not sure what's actually happening in this module, but does this defaultTheme need to be "dark" or "light"? Or should it also be "system"?

Reply:
We've been going with `dark` as the `defaultTheme` across psibase apps.